### DISCUSSION TITLE
How to remove/stop a schedule

### DISCUSSION THREAD

**Discussion by YishuiLi** [PrefectHQ/prefect#4910] [Q&A]
Create a schedule like this:
```
from prefect import task, Flow
from datetime import timedelta
from prefect.schedules import IntervalSchedule

@task
def say_hello():
    print("Hello, world!")

schedule = IntervalSchedule(interval=timedelta(minutes=2))

with Flow("Hello", schedule) as flow:
    say_hello()

flow.run()
```
How to remove/stop this schedule?

**Comment by kvnkho** [comment]
Hey @YishuiLi , if you are using `flow.run()`, you should just remove the schedule from your code and re-run this. 

You can also do `flow.run(run_on_schedule=False)`. Does that answer you?

**Reply by YishuiLi** [reply]
Thank you for your reply! That was helpful.
